media-video/gxine/gxine-0.5.909.ebuild

```diff
@@ -45,22 +50,32 @@ src_prepare() {
 
 src_configure() {
 	econf \
-		$(use_enable nls) \
-		$(use_enable lirc) \
 		--enable-watchdog \
 		--with-xcb \
-		$(has_version '<dev-lang/spidermonkey-1.8.5' && echo --with-spidermonkey=/usr/include/js) \
+		--without-dbus \
+		--without-hal \
+		$(use_enable nls) \
+		$(use_enable lirc) \
 		$(use_with nsplugin browser-plugin) \
 		$(use_with udev gudev) \
-		--without-hal \
-		--without-dbus \
 		$(use_with xinerama)
 }
 
+PLUGINS_DIR="nsbrowser/plugins"
```

Comment:
Skimming - does this need to be in global scope?

Reply:
I removed the nsplugins stuff completely so it's dropped